TRADEMARKS.md

```diff
@@ -0,0 +1,231 @@
+- The AmpliPi software installed on the device is only be modified so far as necessary to operate on the hardware platforms and the essential functions of the software are unchanged. 
+- You do not suggest that the Project is the source of the hardware device itself but rather that the Marks are for the software incorporated into the device. For example, "[Your product name] runs AmpliPi software" or "[Your product name] with AmpliPi software" are acceptable, but "[Your product name] AmpliPi", "[Your product name] AmpliPi controller", or "AmpliPi by [your company name]" are not acceptable.
+- You must release full and unabridged schematics for your hardware device.
+- You must contact us and obtain prior written approval. See section "Where to Get Further Information" for our contact information. This review process helps ensure that hardware products associated with AmpliPi adhere to open source principles, benefit the AmpliPi community and our contributors, and are compliant with the AmpliPi trademark policies, software license, and applicable laws.
+
+## Uses we consider infringing without seeking further permission from us
+
+We will likely consider using the Marks in a software distribution that combines our software with any other software program an infringement of our Marks. We would consider your software "combined" with ours if you create a single binary for both software programs, as well as if by installing our software it automatically installs yours. We will likely not consider your software "combined" with ours if it is on the same media but requires separate, independent action to install it. 
+
+Having said that, if you have new ideas for integrating AmpliPi with other software, please reach out to us for further discussion. We are open to new ideas that are of benefit to the AmpliPi project, our contributors, and to the open source community as a whole. Our contact information can be found in the section "Where to Get Further Information".
+
+# Use for Non-Software Goods and Services
+
+See "Universal Considerations for All Uses", above, which also apply. 
+
+## Uses we consider non-infringing
+
+### Websites
+
+You may use the Word Marks and Logos, but not the Trade Dress, on your webpage to show your support for the Project as long as:
+
+- The website has branding that is easily distinguished from the Project Trade Dress;
+- You own branding or naming is more prominent than any Project Marks;
+- The Logos hyperlink to the Project website;
+- The site does not mislead customers into thinking that either your website, service, or product is our website, service, or product; and
+- The site clearly states that you are not affiliated with or endorsed by the Project.
+
+### Publications and presentations
+
+You can use the Word Marks in book and article titles, and the Logo in illustrations within the document, as long as the use does not suggest that we have published, endorse, or agree with your work. 
```

Comment:
The 'have' in "does not suggest that we have published, endorse, or agree with your work." makes the things in the list (other than published) sound kind of weird. Maybe remove it?

Reply:
What about "_...as long as the use does not suggest that we have published, endorsed, or agreed with your work._"